### PR TITLE
Feature/hero

### PR DESCRIPTION
**Pull request by pananimals**

### CODE REVIEW

File: src/css/styles/_c-banner.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  margin: 0 auto;
+  margin-bottom: 6rem;
+  background: url("../img/c-banner__bg.jpg") no-repeat 40% 60%;
+  background-size: auto 130%;
+
+  &__title {
+    margin-bottom: 3rem;
+    font-size: 4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
```
Comment (michalkowol) [inline src/css/styles/_c-banner.scss]: image

- [ ] colors

File: src/css/styles/_c-banner.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  &__txt {
+    margin-bottom: 4rem;
+    font-size: 1.4rem;
+    line-height: 120%;
```
Comment (michalkowol) [inline src/css/styles/_c-banner.scss]: image

- [ ] 120% vs 150%

File: src/css/styles/_c-banner.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    &__title {
+      margin-bottom: 1rem;
+      font-size: 2rem;
+    }
+
+    &__txt {
+      margin-bottom: 1.5rem;
+      font-size: 0.8rem;
```
Comment (michalkowol) [inline src/css/styles/_c-banner.scss]: image

- [ ] font-size on mobile

File: src/css/styles/_c-banner.scss [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  background-size: auto 130%;
+
+  &__title {
+    margin-bottom: 3rem;
+    font-size: 4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
+  }
+
+  &__txt {
```
Comment (michalkowol) [inline src/css/styles/_c-banner.scss]: - [ ] `c-banner__txt` vs `c-banner__description` (name)

File: src/css/styles/_layout.scss [hunk 5]
```diff
@@ -1,9 +1,3 @@
-.l-container-padding {
-  padding-right: 4rem;
-  padding-left: 4rem;
-
-  @include media-breakpoint-down(md) {
-    padding-right: 2rem;
-    padding-left: 2rem;
-  }
+.l-no-padding {
+  padding: 0;
```
Comment (michalkowol) [inline src/css/styles/_layout.scss]: - [ ] `layout` or `helper`?

File: src/index.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
 import {
-  setMobileMenuFullscreen, toggleMobileMenu, hideMobileMenuForDesktop, hideMobileMenuOnClick
+  setMobileMenuFullscreen, toggleMobileMenu, hideMobileMenuForDesktop, hideMobileMenuOnClick,
+  setupBanner
```
Comment (michalkowol) [inline src/index.js]: I do not find very readable. Should we add `"object-curly-newline": ["error", {"ImportDeclaration": "never"}],` to `.eslintrc`?

- [ ] object-curly-newline

File: src/css/styles/_c-banner.scss (hunk 4 above)
Comment (pananimals) [reply src/css/styles/_c-banner.scss]: Well, I don't mind- if you think it's better, let's go do it this way ;)

File: src/css/styles/_c-banner.scss (hunk 2 above)
Comment (pananimals) [reply src/css/styles/_c-banner.scss]: Yup, looks better. For mobiles it's 140%, above xs it's 150%.

File: src/css/styles/_c-banner.scss (hunk 3 above)
Comment (pananimals) [reply src/css/styles/_c-banner.scss]: Is it 0.9rem? Looks better on most mobiles, on the smallest text part can be slightly too big, but still- it's worth the change. Now on mobiles it's:  `font-size: 0.9rem;` and `line-height: 140%;`

File: src/css/styles/_layout.scss (hunk 5 above)
Comment (pananimals) [reply src/css/styles/_layout.scss]: It's a layout helper. For me can be both `h-*` and `l-*`,

File: src/css/styles/_c-banner.scss (hunk 3 above)
Comment (michalkowol) [reply src/css/styles/_c-banner.scss]: it is 1rem. IMHO 0.9 is still to small

**Pull request merged**